SU-SOE/soe_basic
# D8CORE-3487: fixing the external and internal icons on the links

# READY FOR REVIEW

# Summary
- fixing the icon that comes after the link on the news vertical cards on the news node.
- also corrects it on news cards in a list.

# Review By (Date)
- 2/11

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Look at a news page at the More news. 
4. Notice the internal links have the arrow and the external have the external icon.

# Affected Projects or Products
- Soe News cards

# Associated Issues and/or People
- D8CORE-3487

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
